node/cn/doc.go

```diff
@@ -25,7 +25,7 @@ Source Files
 
   - api.go              : provides private debug API related to block and state
   - api_backend.go      : implements CNAPIBackend which is a wrapper of CN to serve API requests
-  - api_tracer.go       : provides private debug API related to trace chain, block and state
+  - api.go       : provides private debug API related to trace chain, block and state
```

Comment:
We have the explanation of `api.go` two lines above.  The new exploration should be placed at `node/cn/tracers/doc.go`

Reply:
oh.. thank you. Reflected at https://github.com/klaytn/klaytn/pull/1581/commits/c2360ce46cb2c71560c2e0557bad56abed7af9bc